prime-angular-frontend/src/app/modules/site-registration/shared/components/contact-profile-form/contact-profile-form.component.html

```diff
@@ -56,6 +56,17 @@
     </app-page-subheader2>
 
     <div class="row">
+      <div class="col-12">
+
+        <mat-form-field class="w-100">
```

Comment:
Got tired of looking at required, optional, optional, required so moved it so required fields are paired